ports/rp2/mphalport.c

```diff
@@ -44,6 +44,30 @@ ringbuf_t stdin_ringbuf = { stdin_ringbuf_array, sizeof(stdin_ringbuf_array) };
 
 #endif
 
+#if MICROPY_HW_ENABLE_USBDEV
+
+#ifndef USB_BUFFER_LEN
+#define USB_BUFFER_LEN 512
```

Comment:
should start with `MICROPY_HW_`, eg `MICROPY_HW_STDIN_RINGBUF_LEN`

Reply:
done